src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/collectors/ThreadPoolMetricsCollector.java

```diff
@@ -142,32 +140,16 @@ public long getRejected() {
         public final long rejected;
         public final int threadsCount;
         public final int threadsActive;
-        @JsonInclude(Include.NON_NULL)
-        public final Double queueLatency;
-        @JsonInclude(Include.NON_NULL)
-        public final Integer queueCapacity;
-
-          public ThreadPoolStatus(String type,
```

Comment:
Is this constructor no longer used ?  I am assuming we were using this for testing purpose.

Reply:
I checked the codebase and this constructor is not used anywhere.